docs/data-sources/account_team_member.md

```diff
@@ -25,8 +25,8 @@ data "aiven_account_team_member" "foo" {
 
 ### Required
 
-- `account_id` (String) The unique account id This property cannot be changed, doing so forces recreation of the resource.
-- `team_id` (String) An account team id This property cannot be changed, doing so forces recreation of the resource.
+- `account_id` (String) The unique account id. This property cannot be changed, doing so forces recreation of the resource.
+- `team_id` (String) An account team id. This property cannot be changed, doing so forces recreation of the resource.
```

Comment:
Please remove trailing dot. That is 80% of diff. Let's make it in a separate PR and focus on default values now.